src/bot.js

```diff
@@ -69,29 +63,28 @@ class HangoutsChatBot extends Adapter {
       thread = undefined,
       text = '',
       cardString = '[]',
-      httpRes = undefined) {
+      httpRes = undefined,
+      done = ()=>{}) {
     if (text == '' && cardString == '[]') {
```

Comment:
It looks like this callback isn't used except for logging. Could we remove it?

Reply:
The callback is used to execute `res.status(200).end()` dependably. It's in ```        this.onEventReceived(req.body, res, ()=>{
          res.status(200).end();
        });
```. If the only scenario where `res.end()` has to be called is if `resp.reply()` is used, then their should be another way to design this too.

Reply:
Isn't that the callback in `onEventReceived`? This callback is created in `send`/`reply` and is either empty or logging.

Reply:
the callback in the original code that called `this.robot.receive` called res.end. However, in the scenario when multiple scripts handle the same message, this design breaks down.